docs/development.rst

```diff
@@ -10,6 +10,16 @@ This chapter will get you started with |project_name| development.
 |project_name| is written in Python (with a little bit of Cython and C for
 the performance critical parts).
 
+Code and issues
+---------------
+
+Code is stored on `Github <https://github.com/>`_, in the
```

Comment:
maybe do not link to github, this will only lead to people clicking on this link, expecting to see our code there.